subworkflows/local/gtdbtk.nf

```diff
@@ -14,49 +14,25 @@ workflow GTDBTK {
     gtdb_mash      // channel: path
 
     main:
-    // Filter bins: classify only medium & high quality MAGs
-    ch_bin_metrics = Channel.empty()
-    if (params.binqc_tool == 'busco') {
-        // Collect completeness and contamination metrics from busco summary
-        ch_bin_metrics = bin_qc_summary
-            .splitCsv(header: true, sep: '\t')
-            .map { row ->
-                def completeness = -1
-                def contamination = -1
-                def missing
-                def duplicated
-                def busco_db = file(params.busco_db)
-                if (busco_db.getBaseName().contains('odb10')) {
-                    missing = row.'%Missing (specific)'
-                    // TODO or just take '%Complete'?
-                    duplicated = row.'%Complete and duplicated (specific)'
-                }
-                else {
-                    missing = row.'%Missing (domain)'
-                    duplicated = row.'%Complete and duplicated (domain)'
-                }
-                if (missing != '') {
-                    completeness = 100.0 - Double.parseDouble(missing)
-                }
-                if (duplicated != '') {
-                    contamination = Double.parseDouble(duplicated)
-                }
-                [row.'GenomeBin', completeness, contamination]
```

Comment:
Does this not matter anymore because the columns are fixed now in the BUSCO tables?

Reply:
Exactly!